.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
- [ ] Please make your project readme more descriptive.

Reply:
Thanks
I'm forget that!